test/jit/indirect_i64_const_store.wast

```diff
@@ -0,0 +1,11 @@
+(module
+ (memory 1 1)
+ ;; regression test for 32bit JIT
+ (func (export "i64_indirect_store_const") (param i32)
+       (i64.store
+        (local.get 0)
+        (i64.const 0xcafebabe)))
+ (func (export "i64_load") (param i32) (result i64)
+       (i64.load (local.get 0))))
```

Comment:
The last ')' should go to the next line.

Reply:
Old lisp habits. :sweat_smile: 
Should be fixed now.